{"name": "test_two", "status": "broken", "statusDetails": {"message": "TypeError: hasattr expected 2 arguments, got 0", "trace": "self = <jzwpytest.test_pytest.TestDemo object at 0x000001A32E109688>\n\n    def test_two(self):\n        x = \"hello\"\n>       hasattr()\nE       TypeError: hasattr expected 2 arguments, got 0\n\njzwpytest\\test_pytest.py:41: TypeError"}, "attachments": [{"name": "stdout", "source": "bb35beb1-ce21-49ec-bb50-453748a50613-attachment.txt", "type": "text/plain"}], "start": 1589008883136, "stop": 1589008883136, "uuid": "1338b072-8734-4027-8756-4491a9012e45", "historyId": "bdb14a5b80012b58209f2663dddf2438", "testCaseId": "c25ccc5659d466f7e938dba1b29f4520", "fullName": "jzwpytest.test_pytest.TestDemo#test_two", "labels": [{"name": "parentSuite", "value": "jzwpytest"}, {"name": "suite", "value": "test_pytest"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "jiangzhw0100"}, {"name": "thread", "value": "26168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "jzwpytest.test_pytest"}]}